src/app.rs

```diff
@@ -16,9 +102,17 @@ impl Component for App {
 
     fn create(_: Self::Properties, _link: ComponentLink<Self>) -> Self {
         let mut candidates = HashMap::new();
-        let candidate = yozhgoor::candidate();
-        candidates.insert(candidate.slug, candidate);
+        let candidate = (yozhgoor::candidate(), yozhgoor::candidate());
+        let candidate_1 = candidate.0;
+        let candidate_2 = candidate.1;
+
+        let candidate_1_info = CandidateInfo::from_candidate(candidate_1, "yozhgoor");
+        let candidate_2_info = CandidateInfo::from_candidate(candidate_2, "yozhgoor2");
+
+        candidates.insert(candidate_1.slug, candidate_1_info);
+        candidates.insert(candidate_2_info.url, candidate_2_info);
```

Comment:
```suggestion
        let candidate_1_info = CandidateInfo::from_candidate(yozhgoor::candidate(), "yozhgoor");
        let candidate_2_info = CandidateInfo::from_candidate(yozhgoor::candidate(), "yozhgoor2");

        candidates.insert(candidate_1.slug, candidate_1_info);
        candidates.insert(candidate_2_info.url, candidate_2_info);
```

Not sure why you made a tuple just to deconstruct it right after XD